README.md

```diff
@@ -1 +1,183 @@
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [Cryptocurrencies] <a name="about-project"></a>
+
+
+
+**[Cryptocurrencies]** project is an app, that allows users to browse through a list of crypto coins fetched from an api and check out various details about the coins. It's built with React and Redux.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+
+
+
+  <ul>
+    <li><a href="#">JavaScript</a></li>
+  </ul>
+
+
+
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Search_a_coin]**
+- **[Fetch_details]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need node installed.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone git@github.com:abe1able/cryptocurrencies.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd cryptocurrencies
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+``` sh
+    npm run build
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+    npm run test
+```
+
+
+### Deployment
+
+You can deploy this project using netlify.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Abel Seifu**
+
+- GitHub: [@abe1able](https://github.com/abe1able)
+- Twitter: [@AbelSeifu15](https://twitter.com/AbelSeifu15)
+- Linkedin: [@abelseifu](https://www.linkedin.com/in/abel-seifu-184543233/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- **[Remove_coin]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Abe1able/Cryptocurrencies/issues)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, you can show your support by giving it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ]  **Required** 1. The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Resolved, Thank you